language/stdlib/modules/associaton.move

```diff
@@ -0,0 +1,136 @@
+// Implements logic for registering addresses as association addresses, and
+// determining if the sending account is an association account.
+// Errors:
+// 0 -> INVALID_GENESIS_ADDR
```

Comment:
This shouldn't be an issue since we only perform the conversion on errnos for prologue errors. That being said, I think your idea of stratifying these error messages is a good idea so they now start at 1000 and go up.